picoscope/darwin_utils.py

```diff
@@ -14,7 +14,7 @@ def LoadLibraryDarwin(library):
 
     http://ulthiel.com/vk2utl/picoscope-python-interface-under-mac-os-x/
     """
-    PICO_LIB_PATH = "/Applications/PicoScope6.app/Contents/Resources/lib/"
+    PICO_LIB_PATH = "/Applications/PicoScope 6.app/Contents/Resources/lib/"
```

Comment:
Could you please make this more "backward" compatible?? I understand that Picoscope moved the location of their file.

If you could "try both locations" in a loop or something, that would be better.

Reply:
As far as I can see, Picoscope changed the path at least 1.5 years ago, so is there anything to be backwards compatible to?  Is it possible to install a Picoscope library that uses the old path?

Reply:
1.5 years is not a long time for instrumentation. often I'll just leave the software at the version I had when I first got the instrument. it's pretty important to me that the old stuff continues to work at least for a few more months.